File: core/src/com/mygdx/xcube/XCube.java
package com.mygdx.xcube;

import com.badlogic.gdx.graphics.g2d.BitmapFont;
import com.badlogic.gdx.graphics.g2d.SpriteBatch;
import com.badlogic.gdx.Game;
public class XCube extends Game {
	public SpriteBatch batch;
	public BitmapFont font;

	public void create(){
		batch = new SpriteBatch();
		font = new BitmapFont();
		this.setScreen(new MainMenuScreen(this, false, 150f));    // Lance l'écran de départ.

	}


	public void render(){
		super.render();
	}

	public void dispose(){
		batch.dispose();
		font.dispose();
	}
}

File: core/src/com/mygdx/xcube/XCube.kt
package com.mygdx.xcube

import com.badlogic.gdx.Game
import com.badlogic.gdx.graphics.g2d.BitmapFont
import com.badlogic.gdx.graphics.g2d.SpriteBatch

class XCube : Game() {
    var batch: SpriteBatch? = null
    var font: BitmapFont? = null
    override fun create() {
        batch = SpriteBatch()
        font = BitmapFont()
        setScreen(MainMenuScreen(this)) // Lance l'écran de départ.
    }

    override fun render() {
        super.render()
    }

    override fun dispose() {
        batch!!.dispose()
        font!!.dispose()
    }
}